libs/@guardian/source/src/react-components/index.ts

```diff
@@ -121,7 +121,8 @@ export { SvgChevronUpAndDownSmall } from './__generated__/icons/SvgChevronUpAndD
 export { SvgChevronUpDouble } from './__generated__/icons/SvgChevronUpDouble';
 export { SvgChevronUpSingle } from './__generated__/icons/SvgChevronUpSingle';
 export { SvgChevronUpSingleSmall } from './__generated__/icons/SvgChevronUpSingleSmall';
-export { SvgClock } from './__generated__/icons/SvgClock';
+export { SvgClockFilled } from './__generated__/icons/SvgClockFilled';
```

Comment:
Do the other new icons need to be added to this index file?

Reply:
Any exports added to `index.ts` will also need to be added to `index.test.ts`.